stdlib/3/_curses.pyi

```diff
@@ -292,227 +292,171 @@ def use_env(flag: bool) -> None: ...
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ..., tl: _chtype = ..., tr: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ..., tl: _chtype = ..., tr: _chtype = ..., bl: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ..., tl: _chtype = ..., tr: _chtype = ..., bl: _chtype = ..., br: _chtype = ...) -> None: ...
+    @overload
+    def box(self) -> None: ...
+    @overload
+    def box(self, vertch: _chtype = ..., horch: _chtype = ...) -> None: ...
     @overload
     def chgat(self, attr: int) -> None: ...
-
     @overload
     def chgat(self, num: int, attr: int) -> None: ...
-
     @overload
     def chgat(self, y: int, x: int, attr: int) -> None: ...
-
     @overload
     def chgat(self, y: int, x: int, num: int, attr: int) -> None: ...
-
     def clear(self) -> None: ...
-
-    def clearok(self, yes: int): ...
-
+    def clearok(self, yes: int) -> None: ...
     def clrtobot(self) -> None: ...
-
     def clrtoeol(self) -> None: ...
-
     def cursyncup(self) -> None: ...
-
     @overload
     def delch(self) -> None: ...
-
     @overload
-    def delch(self, y: int, x: int) -> None: ...
-
+    def delch(self, y: int = ..., x: int = ...) -> None: ...
     def deleteln(self) -> None: ...
-
     @overload
     def derwin(self, begin_y: int, begin_x: int) -> '_CursesWindow': ...
-
     @overload
     def derwin(self, nlines: int, ncols: int, begin_y: int, begin_x: int) -> '_CursesWindow': ...
-
-    def echochar(self, ch: _chtype, attr: Optional[int]) -> None: ...
-
+    def echochar(self, ch: _chtype, attr: Optional[int] = ...) -> None: ...
     def enclose(self, y: int, x: int) -> bool: ...
-
     def erase(self) -> None: ...
-
     def getbegyx(self) -> Tuple[int, int]: ...
-
     def getbkgd(self) -> Tuple[int, int]: ...
-
-    def getch(self, y: Optional[int], x: Optional[int]) -> _chtype: ...
-
-    def get_wch(self, y: Optional[int], x: Optional[int]) -> _chtype: ...
-
-    def getkey(self, y: Optional[int], x: Optional[int]) -> str: ...
-
-    def getmaxyx(self) -> Tuple['_CursesWindow', int, int]: ...
-
+    @overload
+    def getch(self) -> _chtype: ...
+    @overload
+    def getch(self, y: int, x: int) -> _chtype: ...
+    @overload
+    def get_wch(self) -> _chtype: ...
+    @overload
+    def get_wch(self, y: int, x: int) -> _chtype: ...
+    @overload
+    def getkey(self) -> str: ...
+    @overload
+    def getkey(self, y: int, x: int) -> str: ...
+    def getmaxyx(self) -> Tuple[int, int]: ...
     def getparyx(self) -> Tuple[int, int]: ...
-
-    def getstr(self, y: Optional[int], x: Optional[int]) -> None: ...
-
-    def getyx(self) -> Tuple['_CursesWindow', int, int]: ...
-
+    @overload
+    def getstr(self) -> _chtype: ...
+    @overload
+    def getstr(self, n: int) -> _chtype: ...
+    @overload
```

Comment:
The last two overloads are the same.

Reply:
Oh, must have been distracted. The last one should be with argments x, y and n.